src/app/src/components/FacilityListItems.jsx

```diff
@@ -162,7 +172,7 @@ class FacilityListItems extends Component {
             </Button>
         );
 
-        const backRoute = readOnly ? dashboardListsRoute : listsRoute;
+        const backRoute = isAdminUser ? dashboardListsRoute : listsRoute;
```

Comment:
When an admin is returning to the list view using the in-app "Back to lists" button the selected contributor is being preserved but the new responsibility selection is not

![2022-08-10 15 07 54](https://user-images.githubusercontent.com/17363/184029577-8daee130-9e7b-495e-908e-0a699f459e5a.gif)